src/odemis/dataio/tiff.py

```diff
@@ -830,15 +830,24 @@ def _updateMDFromOME(root, das):
             md[model.MD_AR_HOLE_DIAMETER] = hol
             md[model.MD_AR_FOCUS_DISTANCE] = foc
             md[model.MD_AR_PARABOLA_F] = par
-            try:
-                pol = str(ardata.attrib["Polarization"])
-                posqwp = float(ardata.attrib["QuarterWavePlate"])
-                poslinpol = float(ardata.attrib["LinearPolarizer"])
-                md[model.MD_ARPOL_POLARIZATION] = pol
-                md[model.MD_ARPOL_POS_QWP] = posqwp
-                md[model.MD_ARPOL_POS_LINPOL] = poslinpol
-            except (AttributeError, KeyError, ValueError):
-                pass
+        except (AttributeError, KeyError, ValueError):
+            pass
+
+        # TODO what does .find do? seems to work anyways
```

Comment:
You can check the documentation of ElementTree here: https://docs.python.org/2/library/xml.etree.elementtree.html

Basically, it'll look for the first sub-element which is called "POLData". As it returns `None` if no element is found, when there is no such metadata, the next 3 blocks will fail silently, and the code will end-up doing the right thing.